common/reviews/api/node-library-build.api.ts

```diff
@@ -1,13 +1,9 @@
 export declare function addSuppression(str: string): void;
 
 class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig> {
-  // (undocumented)
```

Comment:
Also, @nickpape-msft is going to fix a problem where warnings aren't failing the Travis build.  It's a switch that is turned on in sp-client but not in web-build-tools.